### PR TITLE
[LIWO-578] Toegankelijkheid page

### PR DESCRIPTION
**Pull request by petergoes**
Adds Toegankelijkheid view

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Home from './views/Home'
 import Maps from './views/Maps'
 import Viewer from './views/Viewer'
 import Combine from './views/Combine'
+import Toegankelijkheid from './views/Toegankelijkheid'
```
Comment (SiggyF) [inline src/router.js:10]: Component names in English for consistency

File: src/router.js [hunk 2]
```diff
@@ -105,6 +106,14 @@ const router = new Router({
       meta: {
         title: 'LIWO - Contact'
       }
+    },
+    {
+      path: '/toegankelijkheid',
```
Comment (SiggyF) [inline src/router.js:111]: urls in English for consistency.

File: src/views/Toegankelijkheid.vue [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        <h2 data-class="section-head">Problemen met toegankelijkheid melden</h2>
+        <p>Heeft u vragen of opmerkingen? Of wilt u een pagina gebruiken op deze website die niet toegankelijk blijkt? Neem dan contact met ons op</p>
+      </section>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Toegankelijkheid'
```
Comment (SiggyF) [inline src/views/Toegankelijkheid.vue:38]: English for consistency.

**Review by SiggyF — APPROVED** [review]
Just a few comments about component naming.

File: src/router.js (hunk 1 above)
Comment (petergoes) [reply src/router.js:10]: The component name now reflects the url, do you want:
a) Keep /toegankelijkheid and have the component be Accessibility
b) Rename the url to /accessibility so it match the component name
c) Leave it as is

**Review by SiggyF — COMMENTED** [review]
Make sure you use semantic release format commit messages `feat(accessibility): accessibility page added` so that release notes can be autogenerated.

**Pull request merged**